plugins/multinet.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: multinet
+spec:
+  version: v0.2.1
+  homepage: https://github.com/k8snetworkplumbingwg/kubectl-multinet
+  shortDescription: "Shows pods' network-status annotation of NPWG multi-net-spec"
```

Comment:
This might be tad bit too long drop 1-2 words to fit into "krew list" cmd.

Reply:
Drop two words